front/migrations/20230413_runs.ts

```diff
@@ -0,0 +1,88 @@
+import { App, Run } from "@app/lib/models";
+import { Sequelize } from "sequelize";
+
+const { CORE_DATABASE_URI } = process.env;
+
+async function main() {
+  const core_sequelize = new Sequelize(CORE_DATABASE_URI as string, {
+    logging: false,
+  });
+  const data = await core_sequelize.query("SELECT * FROM runs");
```

Comment:
Is this too memory-hungry ?

Reply:
Seems fine

Reply:
Yes runs don't have traces information so that's really fine. (if it had been the fully "hydrated" runs with traces this would have been too big (some runs with traces are >8mb))